linera-witty/src/runtime/wasmer/mod.rs

```diff
@@ -160,11 +162,13 @@ pub type ReentrantInstance<'a, UserData> = FunctionEnvMut<'a, Environment<UserDa
 impl<UserData: 'static> Instance for ReentrantInstance<'_, UserData> {
     type Runtime = Wasmer;
     type UserData = UserData;
-    type UserDataReference<'a> = MutexGuard<'a, UserData>
+    type UserDataReference<'a>
+        = MutexGuard<'a, UserData>
     where
         Self::UserData: 'a,
         Self: 'a;
-    type UserDataMutReference<'a> = MutexGuard<'a, UserData>
+    type UserDataMutReference<'a>
+        = MutexGuard<'a, UserData>
```

Comment:
Why are these necessary? Is your rustfmt version different from CI?

Reply:
I have no idea how that happened - I always run the same `cargo fmt` command.

Reply:
Yes yes, I do too, I don't know how this happened.

Reply:
Reverted.